docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
+
+WORKDIR /go/src/github.com/hudl/fargo/
+RUN godep restore
+
+#UNcomment if you want to run tests on master branch.
+#CMD go test -v ./...
```

Comment:
Right now most tests require eureka servers running, this is not ideal, I know. But we want to make it easier to actually run those tests. For now we are trying MVP just to run tests once.

We may remove that comment entirely probably now when you mention that. All instructions are in README.md now.